Dock controls at the bottom of call screen

prprabhu-ms opened the pull request:
# What
Update CallComposite's controls to match design spec:

![image](https://user-images.githubusercontent.com/82062616/120723406-8884d300-c486-11eb-9d37-754ff5ed2506.png)


* Dock at the bottom
* Remove Control bar shadow
* Add Composite shadow

# Why

* Design specs. Looks neater.

# How Tested

visually inspect local storybook.

# Process & policy checklist
<!--- Review the list and check the boxes that apply. -->

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.
<!--- If yes, describe the impact. -->

JamesBurnside commented:
Any chance you want to fix the leave button also to match the figma?
![image](https://user-images.githubusercontent.com/2684369/120712011-f32d1300-c474-11eb-8379-e0a6bb86d988.png)

Do you have a link to the figma you are referencing? This one looks like we should show labels by default also:
https://www.figma.com/file/72qJ03EOQExf12g5JTdWjQ/ACS-UI?node-id=13%3A581

JamesBurnside commented on packages/react-composites/src/composites/CallComposite/styles/Call.styles.ts:
```diff
@@ -8,5 +8,6 @@ export const callContainer = mergeStyles({
   minHeight: '30rem',
-  minWidth: '50rem'
+  minWidth: '50rem',
+  boxShadow: 'rgba(0, 0, 0, 0.133) 0px 1.6px 3.6px 0px, rgba(0, 0, 0, 0.11) 0px 0.3px 0.9px 0px'
```
might not be able to, but any chance this can make use of the theme box shadow value? https://developer.microsoft.com/en-us/fluentui#/styles/web/elevation

prprabhu-ms replied:
Done. Thanks for calling this out. We should be respecting as much of the theming as possible in all our hand-rolled containers for Fluent components.

JamesBurnside reviewed (approved):
Small suggestions

prprabhu-ms commented:
> Any chance you want to fix the leave button also to match the figma? :D
> ![image](https://user-images.githubusercontent.com/2684369/120712011-f32d1300-c474-11eb-8379-e0a6bb86d988.png)
> 
> Do you have a link to the figma you are referencing? This one looks like we should show labels by default also:
> https://www.figma.com/file/72qJ03EOQExf12g5JTdWjQ/ACS-UI?node-id=13%3A581

Added labels and fixed the leave button (I think?).

Thanks for pointing these out.

prprabhu-ms commented:
Especially @alcail since we're working on related stuff.
